### PR TITLE
added Sengupta, et. al resources from AICAS, ISCAS, and CISS 2020-2021

### PR DESCRIPTION
**Pull request by js3ng**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -378,6 +378,9 @@ AAAI Conf. Artificial Intelligence, 2020. [PDF](https://www.aaai.org/Papers/AAAI
 - <a name="Monforte20aicas"></a>Monforte, M., Arriandiaga, A., Glover, A., Bartolozzi, C.,  
 *[Exploiting Event Cameras for Spatio-Temporal Prediction of Fast-Changing Trajectories](https://arxiv.org/pdf/2001.01248)*,  
 IEEE Int. Conf. Artificial Intelligence Circuits and Systems (AICAS), 2020.
+- <a name="Sengupta2021aicas"></a>Sengupta, J. P., Kubendran, R., Neftci, E., Andreou, A. G.,
```
Comment (guillermogb) [inline README.md:381]: Thanks for the updates.
Note that in the keys we only use the last 2 digits of the year 2021 -> 21, so Sengupta21aicas.

File: README.md [hunk 2]
```diff
@@ -378,6 +378,9 @@ AAAI Conf. Artificial Intelligence, 2020. [PDF](https://www.aaai.org/Papers/AAAI
 - <a name="Monforte20aicas"></a>Monforte, M., Arriandiaga, A., Glover, A., Bartolozzi, C.,  
 *[Exploiting Event Cameras for Spatio-Temporal Prediction of Fast-Changing Trajectories](https://arxiv.org/pdf/2001.01248)*,  
 IEEE Int. Conf. Artificial Intelligence Circuits and Systems (AICAS), 2020.
+- <a name="Sengupta2021aicas"></a>Sengupta, J. P., Kubendran, R., Neftci, E., Andreou, A. G.,  
+*[High-Speed, Real-Time, Spike-Based Object Tracking and Path Prediction on Google Edge TPU.](https://par.nsf.gov/servlets/purl/10212648)*
```
Comment (guillermogb) [inline README.md:382]: Need two blank spaces at the end of the line on github to force a new line.

File: README.md [hunk 3]
```diff
@@ -450,7 +453,9 @@ J. Imaging, 2021.
 - <a name="Chiberre21cvprw"></a>Chiberre, P., Perot, E., Sironi, A., Lepetit, V.,  
 *[Detecting Stable Keypoints From Events Through Image Gradient Prediction](https://openaccess.thecvf.com/content/CVPR2021W/EventVision/papers/Chiberre_Detecting_Stable_Keypoints_From_Events_Through_Image_Gradient_Prediction_CVPRW_2021_paper.pdf)*,  
 IEEE Conf. Computer Vision and Pattern Recognition Workshops (CVPRW), 2021. [YouTube](https://youtu.be/Rrkwxp8J18c).
-
+- <a name="Sengupta2021ciss0"></a>Sengupta, J. P., Villemur, M., Andreou, A. G.,
```
Comment (guillermogb) [inline README.md:456]: Sengupta21ciss0

File: README.md [hunk 4]
```diff
@@ -1770,7 +1775,9 @@ British Machine Vision Conf. (BMVC), 2019. [PDF](https://arxiv.org/pdf/1907.0746
 - <a name="Liu19msp"></a>Liu, S.-C., Rueckauer, B., Ceolini, E., Huber, A., Delbruck, T.,  
 *[Event-Driven Sensing for Efficient Perception: Vision and audition algorithms](https://doi.org/10.1109/MSP.2019.2928127)*,  
 IEEE Signal Process. Mag., 36(6):29-37, Nov. 2019.
-
+- <a name="Sengupta2021ciss1"></a>Sengupta, J. P., Villemur, M., Andreou, A. G.,
```
Comment (guillermogb) [inline README.md:1778]: Sengupta21ciss1

File: README.md [hunk 5]
```diff
@@ -2295,7 +2303,9 @@ IEEE Int. Conf. Robotics and Automation (ICRA), 2014. [PDF](http://compneuro.uwa
 - <a name="Graf14CSEDU"></a>Graf, R., King, R., Belbachir, A.,  
 *[Braille Vision Using Braille Display and Bio-inspired Camera](http://dx.doi.org/10.5220/0004949302140219)*,  
 Int. Conf. Computer Supported Education (CSEDU), SCITEPRESS Digital Library, (2014), pp. 214 - 219.
-
+- <a name="Sengupta2021architecture"></a>Sengupta, J. P., Villemur, M., Mendat, D. R., Tognetti, G., Andreou, A. G.,
```
Comment (guillermogb) [inline README.md:2306]: Sengupta21iscas  (we use the conference name or first letters that appear in the doi to reference the publication venue as the last part of the key)

**Review by guillermogb — CHANGES_REQUESTED** [review]

**Comment by js3ng** [comment]
Fixed all the year abbrev and tag names.

File: README.md (hunk 2 above)
Comment (guillermogb) [reply README.md:382]: The two spaces are needed after "TPU", not in the last line after the page numbers

image

**Pull request merged**